clients/admin-ui/src/features/common/Header.tsx

```diff
@@ -59,6 +62,11 @@ const Header: React.FC = () => {
             <QuestionIcon color="gray.700" boxSize={4} />
           </Button>
         </Link>
+        <NextLink href={USER_MANAGEMENT_ROUTE} passHref>
+          <Button size="sm" variant="ghost">
```

Comment:
this works, although I think it might be better to use `IconButton` https://chakra-ui.com/docs/components/icon-button which forces you to add some accessibility labels